tests/test_associations_search.rs

```diff
@@ -0,0 +1,45 @@
+use semsimian::{Predicate, RustSemsimian, TermID};
+use std::{collections::HashSet, path::PathBuf};
+
+#[test]
+#[ignore]
+#[cfg_attr(feature = "ci", ignore)]
+fn test_associations_search() {
+    let mut db_path = PathBuf::new();
+    if let Some(home) = std::env::var_os("HOME") {
+        db_path.push(home);
+        db_path.push(".data/oaklib/phenio.db");
```

Comment:
Does feel kinda weird to have phenio.db as an expected test fixture independent of the benchmark, even if it's ignored for now and even if phenio is the best representation of actual usage.
It's another reminder that we need a minimal version of phenio, perhaps

Reply:
I completely agree!